### DISCUSSION TITLE
Automatically turn on Dynamic Port Forwarding

### DISCUSSION THREAD

**Discussion by betacat2048** [isontheline/pro.webssh.net#1037] [Ideas]
WebSSH uses Dynamic Port Forwarding (DPF) to connect to the intranet server through ssh jumper. But every time I need to connect to an intranet server, I need to first enable the corresponding DPF in Forwardings, and then go back to the Servers list to connect. Considering that access to a specific intranet server is usually completed through a designated jumper, this step can be simplified.

It is hoped that an option can be added to the server configuration to allow users to bind the connection to a specific forwarding (that is, when the SSH/SFTP is turned on, the corresponding DPF will be automatically started, and the corresponding DPF will be closed when it is closed) .

Considering that WebSSH supports multiple windows, the port forwarding field of DPF may also need to be adjusted appropriately so that users can use different DPFs for multiple SSH connections.

**Comment by isontheline** [comment]
Hello @betacat2048 👋

I think you are talking about this issue?
https://github.com/isontheline/pro.webssh.net/issues/577
